packages/hash/frontend/src/pages/playground.page/chips.tsx

```diff
@@ -0,0 +1,120 @@
+        <Chip color="green" label="Chip" variant="outlined" />
+        <Chip color="yellow" label="Chip" variant="outlined" />
+        <Chip color="pink" label="Chip" variant="outlined" />
+        <Chip color="teal" label="Chip" variant="outlined" />
+        <Chip color="mint" label="Chip" variant="outlined" />
+        <Chip color="navy" label="Chip" variant="outlined" />
+      </Stack>
+      <Stack direction="row" alignItems="center" spacing={2} mb={2}>
+        <Chip color="gray" label="Chip" />
+        <Chip color="red" label="Chip" />
+        <Chip color="blue" label="Chip" />
+        <Chip color="orange" label="Chip" />
+        <Chip color="purple" label="Chip" />
+        <Chip color="green" label="Chip" />
+        <Chip color="yellow" label="Chip" />
+        <Chip color="pink" label="Chip" />
+        <Chip color="teal" label="Chip" />
+        <Chip color="mint" label="Chip" />
+        <Chip color="navy" label="Chip" />
+      </Stack>
+      <Stack direction="row" alignItems="center" spacing={2} mb={2}>
+        <Chip
+          size="xs"
+          color="gray"
+          label="Chip"
+          onDelete={() => alert("Delete clicked")}
+        />
+        <Chip
+          size="small"
+          color="red"
+          label="Chip"
+          onDelete={() => alert("Delete clicked")}
+        />
+        <Chip
+          size="large"
+          color="blue"
+          label="Chip"
+          onDelete={() => alert("Delete clicked")}
+        />
+      </Stack>
+      <Stack direction="row" alignItems="center" spacing={2} mb={2}>
+        <Chip
+          size="xs"
+          color="gray"
+          label="Chip"
+          icon={<FontAwesomeIcon icon={faPlus} />}
+        />
+        <Chip
+          size="small"
+          color="red"
+          label="Chip"
+          icon={<FontAwesomeIcon icon={faPlus} />}
+        />
+        <Chip
+          size="large"
+          color="blue"
+          label="Chip"
+          icon={<FontAwesomeIcon icon={faPlus} />}
+        />
+      </Stack>
+      <Stack direction="row" alignItems="center" spacing={2} mb={2}>
+        <Chip color="gray" label="Chip" hasCircleStartIcon />
+        <Chip color="red" label="Chip" hasCircleStartIcon />
+        <Chip color="blue" label="Chip" hasCircleStartIcon />
+      </Stack>
+
+      <Stack direction="row" alignItems="center" spacing={2} mb={2}>
+        <ChipGroup>
+          <Chip color="blue" label="Compound" />
+          <Chip color="orange" label="Chip" />
+        </ChipGroup>
+        <ChipGroup>
+          <Chip color="green" label="Number" />
+          <Chip color="green" label="1-1000" />
+        </ChipGroup>
+        <ChipGroup>
+          <Chip color="purple" label="Text" />
+          <Chip color="purple" label="0-120ch" />
+          <Chip label="Multiple" />
+        </ChipGroup>
+        <ChipGroup>
+          <Chip color="orange" label="Chip" />
+        </ChipGroup>
```

Comment:
A huge chunk of file changes in this PR are due to the examples in the playground page. We wouldn't need this page once we switch to storybook.